src/main/kotlin/de/bund/digitalservice/useid/config/ApplicationProperties.kt

```diff
@@ -11,4 +11,8 @@ import org.springframework.validation.annotation.Validated
 class ApplicationProperties {
     @NotBlank
     lateinit var baseUrl: String
+
+    companion object {
+        const val apiVersionPrefix = "/api/v1"
+    }
```

Comment:
my lack of knowledge: why must it be a companion object?

is it possible to do like this?
![image](https://user-images.githubusercontent.com/9930966/234348758-6739aee4-220e-4280-8cb5-8107966a43cd.png)

Reply:
That's the part I am also unsure about. I started with sth. similar to your approach. Unfortunately, this did not work because we use it assign it a lot to `const val` and we need to assign a constant value to that. We need `const val` because we set it a lot in the Controller annotations where we need constants at compile time. 
 However, we can only add a constant value at the top-level. So, the other way instead of having a companion object that I see would be to add it as a top level constant value in the Application Properties. 
That said, I am very new to Kotlin. Therefore, I have probably missed sth and I am not sure what would be the best way to stay true to Kotlin.

I love the variable name btw ;)

Reply:
thanks for the explanation!